services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HakemusFactory.kt

```diff
@@ -0,0 +1,230 @@
+package fi.hel.haitaton.hanke.factory
+
+import fi.hel.haitaton.hanke.HankeEntity
+import fi.hel.haitaton.hanke.allu.ApplicationStatus
+import fi.hel.haitaton.hanke.allu.CustomerType
+import fi.hel.haitaton.hanke.application.ApplicationArea
+import fi.hel.haitaton.hanke.application.ApplicationRepository
+import fi.hel.haitaton.hanke.application.ApplicationService
+import fi.hel.haitaton.hanke.application.ApplicationType
+import fi.hel.haitaton.hanke.application.PostalAddress
+import fi.hel.haitaton.hanke.asJsonResource
+import fi.hel.haitaton.hanke.factory.ApplicationFactory.Companion.createApplication
+import fi.hel.haitaton.hanke.hakemus.ContactResponse
+import fi.hel.haitaton.hanke.hakemus.CustomerResponse
+import fi.hel.haitaton.hanke.hakemus.CustomerWithContactsResponse
+import fi.hel.haitaton.hanke.hakemus.HakemusDataResponse
+import fi.hel.haitaton.hanke.hakemus.HakemusResponse
+import fi.hel.haitaton.hanke.hakemus.HakemusyhteyshenkiloRepository
+import fi.hel.haitaton.hanke.hakemus.HakemusyhteystietoRepository
+import fi.hel.haitaton.hanke.hakemus.JohtoselvitysHakemusDataResponse
+import fi.hel.haitaton.hanke.profiili.ProfiiliClient
+import java.time.ZonedDateTime
+import java.util.*
+import org.geojson.Polygon
+import org.springframework.stereotype.Component
+
+@Component
+class HakemusFactory(
+    private val applicationService: ApplicationService,
+    private val profiiliClient: ProfiiliClient,
+    private val applicationRepository: ApplicationRepository,
+    private val hakemusyhteystietoRepository: HakemusyhteystietoRepository,
+    private val hakemusyhteyshenkiloRepository: HakemusyhteyshenkiloRepository,
+    private val hankeKayttajaFactory: HankeKayttajaFactory,
+) {
+    companion object {
+        private const val DEFAULT_APPLICATION_NAME: String = "Johtoselvityksen oletusnimi"
+        private const val TEPPO_EMAIL = "teppo@example.test"
```

Comment:
You can use the values directly from ApplicationFactory.

Reply:
Use those.